src/v/raft/tests/membership_test.cc

```diff
@@ -25,37 +26,15 @@ FIXTURE_TEST(add_one_node_to_single_node_cluster, raft_test_fixture) {
     auto new_node = gr.create_new_node(model::node_id(2));
     res = retry_with_leader(gr, 5, 1s, [new_node](raft_node& leader) {
               return leader.consensus
-                ->add_group_members({new_node}, model::revision_id(0))
+                ->add_group_member(
+                  raft::vnode(new_node.id(), model::revision_id(0)),
+                  model::revision_id(0))
                 .then([](std::error_code ec) { return !ec; });
           }).get0();
 
     validate_logs_replication(gr);
     auto& leader = gr.get_member(gr.get_leader_id().value());
-    BOOST_REQUIRE_EQUAL(leader.consensus->config().brokers().size(), 2);
-};
-
-FIXTURE_TEST(add_two_nodes_to_the_cluster, raft_test_fixture) {
```

Comment:
nit: we can probably preserve multi-node tests by rewriting them to use `replace_configuration`

Reply:
i will add them in follow up PR